src/panels/config/server_control/ha-config-server-control.ts

```diff
@@ -25,7 +25,7 @@ import { checkCoreConfig } from "../../../data/core";
 const reloadableDomains = [
   "group",
   "automation",
-  "scripts",
```

Comment:
We can prevent this here and in other places in our code by making a `domainType` in our code that matches all valid domains. Not sure if it's worth it.